mono/metadata/marshal.c

```diff
@@ -3967,6 +3968,10 @@ mono_marshal_get_managed_wrapper (MonoMethod *method, MonoClass *delegate_klass,
 
 	mono_marshal_emit_managed_wrapper (mb, invoke_sig, mspecs, &m, method, target_handle);
 
+	if (is_ok (m.error)) {
+		return NULL;
+	}
```

Comment:
What does is_ok do? Are you sure this is not inverted. It is like we are saying, is this error code OK? Cool return NULL.

Reply:
Fixed